packages/suite/src/hooks/settings/backends/useBackendsForm.ts

```diff
@@ -119,22 +120,28 @@ export const useBackendsForm = (coin: Network['symbol']) => {
 
     const input = useBackendUrlInput(coin, currentValues.type, currentValues.urls);
 
-    const save = () => {
-        const { type, urls } = currentValues;
+    const getUrls = () => {
         const lastUrl = input.value && !input.error ? [input.value] : [];
-        const defaultType = getDefaultBackendType(coin);
+        return currentValues.urls.concat(lastUrl);
+    };
 
+    const hasOnlyOnions = () => getUrls().every(isOnionUrl);
```

Comment:
This warns user about missing Tor even if there is no url at all, because all urls from the empty set are indeed onion.
@matejkriz